frontend/src/sass/main.scss

```diff
@@ -128,7 +128,7 @@
 
 .large-input {
   max-width: 600px;
-  min-width: 300px;
+  min-width: 180px;
```

Comment:
what is this change about?

Reply:
Is only used for dns-provider and hibernation schedules days selection. Reduced the min with to make more space in case input fields are empty, DNS Provider row would take up two lines just to show empty input fields for include / exclude input fields.
max-width is unchanged so nothing changes in case content is larger than 300px (which is the case for default hibernation schedules with weekdays in selection)